src/frontend/components/header/header.ts

```diff
@@ -40,12 +40,22 @@ export class Header {
     }
   }
 
+  resetTheme() {
+    this.settingOpened = false;
+    document.removeEventListener('click', this.resetTheme.bind(this), true);
+  }
+
+  openSettings() {
+    document.addEventListener('click', this.resetTheme.bind(this), true);
```

Comment:
@sumitarora why do we need to add an event listener for a click? I would think that angular's `(click)=...` in the template will take care of this? Am I missing something :)